### PR TITLE
Adds a Split/combine pipeline template

### PR DESCRIPTION
**Pull request by tybritten**

### CODE REVIEW

File: pipeline-templates/split-combine/splitcombine.go [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    })
+    if err != nil {
+        return nil, err
+    }
+    sort.Strings(files)
+    return files, nil
+}
+
+
+func splitFile(inputFile string, outputFolder string, lineCount int) error {
```
Comment (BOsterbuhr) [inline pipeline-templates/split-combine/splitcombine.go:90]: Is this only for `csv` files, or will this handle `txt`, `rtf`, and `tex`, for example? And does it only split on new lines?

Comment (tybritten) [reply pipeline-templates/split-combine/splitcombine.go:90]: any text format. splits on newlines only. anything else you'd like to see?

Comment (BOsterbuhr) [reply pipeline-templates/split-combine/splitcombine.go:90]: Splitting on `,` or ` ` could be nice but not completely necessary. No concrete use cases come to mind though.

**Review by BOsterbuhr — APPROVED** [review]
Nice work! A thought to think about, should we add any details about how others could contribute to the pipeline-templates directory and what the requirements would be?

**Pull request merged**